Sources/Testing/Support/FileHandle.swift

```diff
@@ -0,0 +1,224 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+internal import TestingInternals
+
+#if !SWT_NO_FILE_IO
+/// A type representing a file handle.
+///
+/// Instances of this type abstract away the C `FILE *` type for Swift.
+///
+/// Whether or not a particular C file handle is safe to use concurrently is
+/// platform- and file-specific. The testing library assumes that the standard
```

Comment:
A space after `file-` would be nice to be clearer that `specific` applies to both `file` and `platform`.

Reply:
This is grammatically correct and I will die on this hill!